src/main/java/com/exadel/demo/service/BookingService.java

```diff
@@ -16,6 +18,9 @@ public class BookingService {
 
     private final ModelMapper modelMapper;
 
+    @Autowired
+    private PublishingMessage publishingMessage;
```

Comment:
Better to use: private final PublishingMessage publishingMessage; like for ModelMapper